src/main/resources/application.properties

```diff
@@ -6,7 +6,7 @@ spring.jackson.serialization.WRITE_DATES_AS_TIMESTAMPS=false
 spring.liquibase.enabled=false
 spring.profiles.active=google
 springfox.documentation.swagger.v2.path=/api-docs
-db.migrate.dropAllOnStart=false
+db.migrate.dropAllOnStart=true
```

Comment:
Okay to modify on your local environment, but this change should not be merged to develop.